File: src/targets/r/httr/fixtures/headers.r
library(httr)

<<<<<<< HEAD
url <- "https://httpbin.org/headers"

response <- VERB("GET", url, add_headers('x-foo' = 'Bar', 'x-bar' = 'Foo'), content_type("application/octet-stream"), accept("application/json"))
=======
url <- "http://mockbin.com/har"

response <- VERB("GET", url, add_headers('x-foo' = 'Bar', 'quoted-value' = '"quoted" \'string\''), content_type("application/octet-stream"), accept("application/json"))
>>>>>>> upstream/master

content(response, "text")

File: src/targets/python/requests/fixtures/headers.py
import requests

<<<<<<< HEAD
url = "https://httpbin.org/headers"
=======
url = "http://mockbin.com/har"
>>>>>>> upstream/master

headers = {
    "accept": "application/json",
    "x-foo": "Bar",
<<<<<<< HEAD
    "x-bar": "Foo"
=======
    "quoted-value": "\"quoted\" 'string'"
>>>>>>> upstream/master
}

response = requests.get(url, headers=headers)

<<<<<<< HEAD
print(response.text)
=======
print(response.json())
>>>>>>> upstream/master
